src/restaurant_inspections_api/constants.clj

```diff
@@ -6,3 +6,4 @@
 (def production? (env/production?))
 (def db-url (env/get-env-db-url))
 (def csv-files (env/get-csv-files))
+(def *earliest-date* "2013-01-01")
```

Comment:
for non-constants that we can instantiate temporarily, we use `let`. For variables that will definitely change (that define state or are going to be changed on the program) clojure includes `atoms` and `refs` :
http://www.braveclojure.com/zombie-metaphysics/